recipes/dlo-hic-tools/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+    name: dlo_hic
+    version: "0.0.0"
+
+source:
+    url: https://github.com/Nanguage/DLO-HiC-Tools/archive/v0.0.0.tar.gz # [unix and py3k]
+    sha256: 5caddccd25fb4f86689dc8b6e85f4eb4a3de990766ddb47670097671cb403b71 # [unix and py3k]
+
+build:
+    number: 1
+    skip: True # [py27]
+
+requirements:
+    host:
+        - python
+        - cython
+        - setuptools
+    run:
```

Comment:
Please make sure to also include the version constraints from https://github.com/Nanguage/DLO-HiC-Tools/blob/v0.0.0/setup.py#L15-L23